packages/css/src/hero/README.md

```diff
@@ -0,0 +1,16 @@
+# Hero
+
+Het hero component is een opvallende tekst aan het begin van een pagina, direct na de header.
+Gebruik dit component om een kernboodschap over te brengen of duidelijk te maken waar de pagina over gaat.
+
+## Richtlijnen
+
+- Gebruik dit component om de aandacht van de gebruiker te trekken.
+- Gebruik dit component maar 1 keer per pagina, vaak alleen op de voorpagina.
+- Niet gebruiken als je lange stukken tekst wilt weergeven, de tekst moet kort en duidelijk zijn.
+
+## Relevante WCAG eisen
+
+Het hero component is een variant van een heading level 1, met een afwijkende stijl. Let bij gebruik van dit component goed op dat de heading hiërarchie van de pagina logisch blijft.
+
+Voor dit component gelden dezelfde WCAG eisen als voor [het heading component](/docs/heading--docs).
```

Comment:
This link seems to have changed. Does a more robust approach to link between docs files exist?

```suggestion
Voor dit component gelden dezelfde WCAG eisen als voor [het heading component](/docs/react_heading--docs).
```

Reply:
Ah, yes, I’m generally using the React Storybook only.